ipn/ipnlocal/tailfs.go

```diff
@@ -64,11 +65,11 @@ func (b *LocalBackend) TailfsSetFileServerAddr(addr string) error {
 	return nil
 }
 
-// TailfsAddShare adds the given share if no share with that name exists, or
+// TailFSAddShare adds the given share if no share with that name exists, or
 // replaces the existing share if one with the same name already exists.
 // To avoid potential incompatibilities across file systems, share names are
```

Comment:
seems like this sentence should move to the Share type's name field?

Reply:
No, this restriction is only enforced here.